shared-pipeline-components/src/__main__.py

```diff
@@ -0,0 +1,3 @@
+from .aiplatform import export_model
+
+export_model.component_spec.save("./export_model.yaml")
```

Comment:
Not seen this before - interesting! This vs calling compile() ?

Reply:
that's me trying out the KFP API! not required here but you can basically compile any component into yaml format which would be a clear interface before importing them in pipelines.